packages/protocol/contracts/identity/interfaces/IFederatedAttestations.sol

```diff
@@ -1,8 +1,30 @@
 pragma solidity ^0.5.13;
 
-// TODO ASv2 add external, view, and only owner function sigs
-// separated into these three groups for clarity
 interface IFederatedAttestations {
+  function registerAttestationAsIssuer(
+    bytes32 identifier,
+    address issuer,
+    address account,
+    uint64 issuedOn
+  ) external;
+  function registerAttestation(
+    bytes32 identifier,
+    address issuer,
+    address account,
+    address signer,
+    uint64 issuedOn,
+    uint8 v,
+    bytes32 r,
+    bytes32 s
+  ) external;
+  function revokeAttestation(bytes32, address, address) external;
+  function batchRevokeAttestations(
+    address issuer,
+    bytes32[] calldata identifiers,
+    address[] calldata accounts
+  ) external;
+
+  // view functions
   function lookupAttestations(bytes32, address[] calldata)
```

Comment:
depending on whether we go with named parameters or not, let's make all the functions in the file consistent; reminder to possibly add param names to the lookup functions